### ISSUE TITLE
cell too big on insertion if full option is false

### ISSUE THREAD

**Issue by tourniayrem** [zzxming/quill-table-up#48]
If 'full' option is set to false, i got this weird behaviour: 
![image](https://github.com/user-attachments/assets/6414ede3-5209-4f45-8c49-18c3bbfa5dfc)
![image](https://github.com/user-attachments/assets/d591c761-9e40-4acf-a3b4-18c8d6a7c1d4)
![image](https://github.com/user-attachments/assets/48816c04-ed86-4486-9e46-768a2c12a6ec)

As you can see, the inserted cell has the same width as the editor, resulting in a scrollable cell just for some pixels

**Version used:**
quill: 2.0.3
quill-table-up: 2.0.1
angular: 18.2.12

**Comment by zzxming** [comment]
I am also confused about this problem. It will display normally under some sizes, but sometimes scroll bars will appear, which I think may be related to the css property of table-collapse. I'm still trying to figure out how to solve this perfectly

**Comment by tourniayrem** [comment]
I'm sure there's a better way, but something like this solves the problem:
```
const colWidth = !this.options.full ? `${Math.max(Math.floor(width / columns), tableUpSize.colMinWidthPx) - 1}px` : `${Math.max((1 / columns) * 100, tableUpSize.colMinWidthPre)}%`;
```

**Comment by zzxming** [comment]
> I'm sure there's a better way, but something like this solves the problem:
> 
> ```
> 
> const colWidth = !this.options.full ? `${Math.max(Math.floor(width / columns), tableUpSize.colMinWidthPx) - 1}px` : `${Math.max((1 / columns) * 100, tableUpSize.colMinWidthPre)}%`;
> 
> ```

I'll think about it and do some test

**Comment by zzxming** [comment]
close with #52 . released version 2.0.2
